amplipi/updater/static/index.html

```diff
@@ -69,12 +71,14 @@ <h1><span class="text-white">Ampli</span><span class="text-danger">Pi</span></h1
                  onclick="ui_start_software_update($('#older-update-sel').val(), $('#older-update-sel').find(':selected').data('version'));">
                 Start Update</a>
             </div>
+
             <div class="tab-pane fade" id="manual-update" role="tabpanel" aria-labelledby="manual-update-tab">
               <div class="custom-file">
                 <input id="update-file-selector" type="file" accept=".gz" onchange="$('#submit-custom-update').removeClass('disabled');">
               </div>
               <a id="submit-custom-update" role="button" class="btn btn-lg btn-primary text-center mt-3 disabled" onclick="ui_upload_software_update();">Start Update</a>
             </div>
+
```

Comment:
Added spacing between divs to make them human readable, my eyes simply couldn't differentiate the sections at a glance otherwise